vendor/ParsedownModern.php

```diff
@@ -15,1524 +15,1524 @@
+	}
+
+	#
+	# Deprecated Methods
+	#
+
+	function parse($text)
+	{
+		$markup = $this->text($text);
+
+		return $markup;
+	}
+
+	#
+	# Static Methods
+	#
+
+	static function instance($name = 'default')
+	{
+		if (isset(self::$instances[$name]))
+		{
+			return self::$instances[$name];
+		}
+
+		$instance = new static();
+
+		self::$instances[$name] = $instance;
+
+		return $instance;
+	}
+
+	private static $instances = array();
+
+	#
+	# Fields
+	#
 
-    protected $DefinitionData;
+	protected $DefinitionData;
 
-    #
-    # Read-Only
+	#
+	# Read-Only
 
-    protected $specialCharacters = array(
-        '\\', '`', '*', '_', '{', '}', '[', ']', '(', ')', '>', '#', '+', '-', '.', '!', '|',
-    );
+	protected $specialCharacters = array(
+		'\\', '`', '*', '_', '{', '}', '[', ']', '(', ')', '>', '#', '+', '-', '.', '!', '|',
+	);
 
-    protected $StrongRegex = array(
-        '*' => '/^[*]{2}((?:\\\\\*|[^*]|[*][^*]*[*])+?)[*]{2}(?![*])/s',
-        '_' => '/^__((?:\\\\_|[^_]|_[^_]*_)+?)__(?!_)/us',
-    );
+	protected $StrongRegex = array(
+		'*' => '/^[*]{2}((?:\\\\\*|[^*]|[*][^*]*[*])+?)[*]{2}(?![*])/s',
+		'_' => '/^__((?:\\\\_|[^_]|_[^_]*_)+?)__(?!_)/us',
+	);
 
-    protected $EmRegex = array(
-        '*' => '/^[*]((?:\\\\\*|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*])/s',
-        '_' => '/^_((?:\\\\_|[^_]|__[^_]*__)+?)_(?!_)\b/us',
-    );
+	protected $EmRegex = array(
+		'*' => '/^[*]((?:\\\\\*|[^*]|[*][*][^*]+?[*][*])+?)[*](?![*])/s',
+		'_' => '/^_((?:\\\\_|[^_]|__[^_]*__)+?)_(?!_)\b/us',
+	);
 
-    protected $regexHtmlAttribute = '[a-zA-Z_:][\w:.-]*(?:\s*=\s*(?:[^"\'=<>`\s]+|"[^"]*"|\'[^\']*\'))?';
+	protected $regexHtmlAttribute = '[a-zA-Z_:][\w:.-]*(?:\s*=\s*(?:[^"\'=<>`\s]+|"[^"]*"|\'[^\']*\'))?';
 
-    protected $voidElements = array(
-        'area', 'base', 'br', 'col', 'command', 'embed', 'hr', 'img', 'input', 'link', 'meta', 'param', 'source',
-    );
+	protected $voidElements = array(
+		'area', 'base', 'br', 'col', 'command', 'embed', 'hr', 'img', 'input', 'link', 'meta', 'param', 'source',
+	);
 
-    protected $textLevelElements = array(
-        'a', 'br', 'bdo', 'abbr', 'blink', 'nextid', 'acronym', 'basefont',
-        'b', 'em', 'big', 'cite', 'small', 'spacer', 'listing',
-        'i', 'rp', 'del', 'code',          'strike', 'marquee',
-        'q', 'rt', 'ins', 'font',          'strong',
-        's', 'tt', 'sub', 'mark',
-        'u', 'xm', 'sup', 'nobr',
-                   'var', 'ruby',
-                   'wbr', 'span',
-                          'time',
-    );
-}
\ No newline at end of file
+	protected $textLevelElements = array(
+		'a', 'br', 'bdo', 'abbr', 'blink', 'nextid', 'acronym', 'basefont',
+		'b', 'em', 'big', 'cite', 'small', 'spacer', 'listing',
+		'i', 'rp', 'del', 'code',          'strike', 'marquee',
+		'q', 'rt', 'ins', 'font',          'strong',
+		's', 'tt', 'sub', 'mark',
+		'u', 'xm', 'sup', 'nobr',
+		           'var', 'ruby',
+		           'wbr', 'span',
+		                  'time',
+	);
```

Comment:
These artistic lines is the only place where TAB-s are continues with spaces.
May I list those elements vertically?